src/services/notifications.service.js

```diff
@@ -176,50 +156,69 @@ const sendEmailOfType = async (
     );
     throw err;
   }
-  return emailSent;
+  statusEmitter.emit("incrementCount", "emailNotificationsSucceeded");
```

Comment:
Each of these statements could just be moved back inside try block for consistency

Reply:
done